rotordc_mail_activity_display_info/static/src/xml/activity.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates xml:space="preserve">
+    <t t-extend="mail.activity_items">
+        <!--
+        FIXME: It would be really nice if we could surgically add/remove
+        classes instead of copy-pasting an entire thing.
+        -->
+        <t t-jquery="a.o_activity_info" t-operation="attributes">
+            <!-- remove 'collapsed' -->
+            <attribute
+                name="class"
+            >btn btn-link btn-info text-muted o_activity_info ml4</attribute>
+            <attribute name="aria-expanded">true</attribute>
+        </t>
```

Comment:
Have you tried to do it with this method? (I guess it works for templates?)
(and same for the one below adding 'show')
```
<xpath expr="//div[@id='node_id']" position="attributes">
    <attribute name="class" separator=" " add="d-none" />
</xpath>
```
from the link you sent me https://dev.to/opencode/don-t-break-your-and-others-odoo-xml-373i ^^

Reply:
I know, but this is a Python dictionary :( I have to remove the closing bracket, add the key-value pair I want, and re-add the closing bracket.

Reply:
Tried this; reverted in [aebb624](https://github.com/coopiteasy/cie-custom/pull/72/commits/aebb6247ab458ff9061a012b4d29b09c6ff41aa1)

It didn't work.